packages/databricks-vscode/src/test/e2e/init.e2e.ts

```diff
@@ -0,0 +1,118 @@
+import assert from "node:assert";
+import {
+    dismissNotifications,
+    waitForInput,
+    getTabByTitle,
+    getViewSection,
+} from "./utils.ts";
+import {sleep, Workbench} from "wdio-vscode-service";
+import {Key} from "webdriverio";
+
+describe("Configure Databricks Extension", async function () {
+    // this will be populated by the tests
+    let workbench: Workbench;
+    // let projectDir: string;
+
+    this.timeout(3 * 60 * 1000);
+
+    before(async function () {
+        assert(
+            process.env.TEST_DEFAULT_CLUSTER_ID,
+            "TEST_DEFAULT_CLUSTER_ID env var doesn't exist"
+        );
+        assert(
+            process.env.WORKSPACE_PATH,
+            "WORKSPACE_PATH env var doesn't exist"
+        );
+        assert(
+            process.env.DATABRICKS_HOST,
+            "DATABRICKS_HOST env var doesn't exist"
+        );
+        // projectDir = process.env.WORKSPACE_PATH;
+        workbench = await browser.getWorkbench();
+        await dismissNotifications();
+    });
+
+    it("should open VSCode", async function () {
+        const title = await workbench.getTitleBar().getTitle();
+        assert(title.indexOf("[Extension Development Host]") >= 0);
+    });
+
+    it("should dismiss notifications", async function () {
+        //Collect all notifications
+        sleep(2000);
+        const notifications = await workbench.getNotifications();
+        for (const n of notifications) {
+            await n.dismiss();
+        }
+    });
+
+    it("should wait for quickstart or changelog", async () => {
+        const section = await getViewSection("CONFIGURATION");
+        assert(section);
+        const welcome = await section.findWelcomeContent();
+        assert(welcome);
+        try {
+            await getTabByTitle("DATABRICKS.quickstart.md");
+        } catch (error) {
+            console.log(
+                "Failed to find quickstart tab, searching for a changelog instead"
+            );
+            await getTabByTitle("Preview CHANGELOG.md");
```

Comment:
We might show "What's new" as well. Do we really need to wait for this?

Reply:
After we've moved to persistent login quick picks we don't have to wait for the changelog tab anymore, as it shouldn't close the quickpick. Right now it's here just to check the actual fact that we show those things. But yeah, I better change it, since we can show other things too